DS-Assignment-4/question2/InvalidInput.java

```diff
@@ -0,0 +1,14 @@
+package question2;
+
+/**
+ * InvalidInput is error class which will be called when user will enter null
+ * string
+ * 
+ * @author Arjita
+ *
+ */
+public class InvalidInput extends Exception {

```

Comment:
We don't require to create same class for 2 different questions. Can import the Exception class from previous question for this question